lua/lvim/config/init.lua

```diff
@@ -52,6 +52,10 @@ function M:init()
     footer = {},
   }
 
+  lvim.builtin.friendly_snippets = {
+    active = true,
+  }
```

Comment:
```suggestion
  lvim.builtin.luasnip = {
    sources = {
      friendly_snippets = true
    }
  }
```

Reply:
I think it's better to make these specific to luasnip instead, because there're certain settings that can probably be exposed.

Reply:
That makes sense 🤔

Reply:
I think that ultimately, we'd want to create a separate file for luasnip, that follows the same pattern as all the others builtins, where all of these settings could live
https://github.com/emersonmx/LunarVim/blob/764fd7dbadf962395950fa01635b394aa5b60b60/lua/lvim/plugins.lua#L72-L89
that would be useful for: 
- allow choosing different directories for storing the snippets
- adding some internal `lvim` snippets that could be useful, e.g. for use in `config.lua` 😄 

cc: @abzcoding

Reply:
@kylo252 do we merge this one for now or ?

Reply:
sorry for the delay! 

@emersonmx, please fix those two small changes, so we can merge this.